napari/layers/points/points.py

```diff
@@ -1270,6 +1272,144 @@ def _get_value(self, position) -> Union[None, int]:
 
         return selection
 
+    def _get_value_3d(
+        self,
+        start_point: np.ndarray,
+        end_point: np.ndarray,
+        dims_displayed: List[int],
+    ) -> Union[int, None]:
+        """Get the layer data value along a ray
+
+        Parameters
+        ----------
+        start_point : np.ndarray
+            The start position of the ray used to interrogate the data.
+        end_point : np.ndarray
+            The end position of the ray used to interrogate the data.
+        dims_displayed : List[int]
+            The indices of the dimensions currently displayed in the Viewer.
+
+        Returns
+        -------
+        value : Union[int, None]
+            The data value along the supplied ray.
+        """
+        if (start_point is None) or (end_point is None):
+            # if the ray doesn't intersect the data volume, no points could have been intersected
+            return None
+        plane_point, plane_normal = click_plane_from_intersection_points(
+            start_point, end_point, dims_displayed
+        )
+
+        # project the in view points onto the plane
+        projected_points, projection_distances = project_points_onto_plane(
+            points=self._view_data,
+            plane_point=plane_point,
+            plane_normal=plane_normal,
+        )
+
+        # rotate points and plane to be axis aligned with normal [0, 0, 1]
+        rotated_points, rotation_matrix = rotate_points_on_plane(
+            points=projected_points,
+            current_plane_normal=plane_normal,
+            new_plane_normal=[0, 0, 1],
+        )
+        rotated_click_point = np.dot(rotation_matrix, plane_point)
+
+        # find the points the click intersects
+        distances = abs(rotated_points[:, :2] - rotated_click_point[:2])
+        in_slice_matches = np.all(
+            distances <= np.expand_dims(self._view_size, axis=1) / 2,
+            axis=1,
+        )
+        indices = np.where(in_slice_matches)[0]
+
+        if len(indices) > 0:
+            # find the point that is most in the foreground
+            candidate_point_distances = projection_distances[indices]
+            closest_index = indices[np.argmin(candidate_point_distances)]
+            selection = self._indices_view[closest_index]
+        else:
+            selection = None
+        return selection
+
+    def _display_bounding_box_augmented(self, dims_displayed: np.ndarray):
```

Comment:
Same comment I had elsewhere: this method name is a bit confusing to me, since it sounds like a verb ("display this!"). Same is true of the base layer method!

Reply:
I can see how that would be confusing. Do you have a name suggestion? just `bounding_box_augmented`?

Reply:
I think `displayed_bounding_box` or `scene_bounding_box` would be good BUT this method is mirroring the (not well named) property on the base layer.

In general we don't have a consistent way of describing the 'displayed_world' coordinate system, usually called `scene` in other programs, and should think about this a bit but I think for the purposes of this PR this name is correct and we should refactor separately. Thoughts?

Reply:
Yeah, sounds good to me. I do think both this and the base layer one should be renamed, but we can keep it for another PR :)